sphinx/domains/python.py

```diff
@@ -833,7 +833,7 @@ def resolve_xref(self, env, fromdocname, builder,
         elif len(matches) > 1:
             logger.warning('more than one target found for cross-reference %r: %s',
                            target, ', '.join(match[0] for match in matches),
-                           location=node)
+                           type='ref', subtype='python', location=node)
```

Comment:
I feel `ref.python` is too much wide name for warning types. any ideas?
It might be better to use more categorized settings (like `ref.python.func`).